OADP_476:Where to get Velero CLI tool [new PR] (#45926)
openshift/openshift-docs: modules/velero-obtaining-by-accessing-binary.adoc

// Module included in the following assemblies:
//
// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc

:_content-type: PROCEDURE
[id="velero-obtaining-by-accessing-binary_{context}"]
= Accessing the Velero binary in the Velero deployment in the cluster

You can use a shell command to access the Velero binary in the Velero deployment in the cluster.

.Prerequisites

* our Data Protection Application custom resource has a status of `Reconcile complete`.

> .Procedure

Reviewer: what's the procedure part? seems like it has a wrong context

Author: Fixed; thanks.

// Module included in the following assemblies:
//
// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc

:_content-type: PROCEDURE
[id="velero-obtaining-by-accessing-binary_{context}"]
= Accessing the Velero binary in the Velero deployment in the cluster

You can use a shell command to access the Velero binary in the Velero deployment in the cluster.

.Prerequisites

> * our Data Protection Application custom resource has a status of `Reconcile complete`.

Reviewer 2: Looks like you're missing a letter here. 

```suggestion
* Your Data Protection Application custom resource has a status of `Reconcile complete`.
```

Reviewer 2: @kaovilai Capitalization differs based on usage: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reviewer 3: ```suggestion
* Your DataProtectionApplication custom resource has a status of `Reconcile complete`.
```
Ok. then remove spacing to clarify this is k8s object reference.

Reviewer 2: ```suggestion
* our `DataProtectionApplication` custom resource has a status of `Reconcile complete`.
```

Author: Yes, I deleted the file you mentioned. it was empty.  Formatting changes made according to final comment.

.Procedure

> * Set the following alias:

Author: Done.
